src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -533,3 +533,17 @@ EquationViewModel ^ EquationInputArea::GetViewModelFromEquationTextBox(Object ^
 
     return eq;
 }
+
+ SolidColorBrush ^ EquationInputArea::GetForegroundColor(Color lineColor)
```

Comment:
Nit: can you create an helper in Utils instead so we can use it in other controls if necessary?

Reply:
Good idea, we may want to add a comment that we don't take alpha into account just in case